esmvalcore/config-developer.yml

```
@@ -127,63 +127,63 @@ CMIP5:
 OBS:
   cmor_strict: false
   input_dir:
-    default: 'Tier[tier]/[dataset]'
-    BSC: '[type]/[institute.lower]/[dataset.lower]/[freq_folder]/[short_name][freq_base]'
-    RCAST: '[dataset]'
+    default: 'Tier{tier}/{dataset}'
+    BSC: '{type}/{institute.lower}/{dataset.lower}/{freq_folder}/{short_name}{freq_base}'
+    RCAST: '{dataset}'
   input_file:
-    default: '[project]_[dataset]_[type]_[version]_[mip]_[short_name]_*.nc'
-    BSC: '[short_name]_*.nc'
-    RCAST: '[short_name]_[mip]_[type]_[dataset]_*.nc'
+    default: '{project}_{dataset}_{type}_{version}_{mip}_{short_name}_*.nc'
```

```suggestion
    default: '{project}_{dataset}_{type}_{version}_{mip}_{short_name}[_]*.nc'
```
This would make the `_` optional, right?

---

See discussion in #170

---

Yes, it should be 
```yaml
    default: '{project}_{dataset}_{type}_{version}_{mip}_{short_name}[._]*nc'
```
so it matches files with and without dates in the name, but not `short_name`s starting with the same characters, e.g. `ta` and `tas`.